CONTRIBUTING.md

```diff
@@ -28,22 +48,6 @@ make test-with-docker
 
 ## Integration Tests
 
-Before running the integration tests, please ensure that the environment variables are set up.
-
-```sh
-export NOTIFY_API_URL="https://example.notify-api.url"
-export API_KEY="example_API_test_key"
-export FUNCTIONAL_TEST_NUMBER="valid mobile number"
-export FUNCTIONAL_TEST_EMAIL="valid email address"
-export EMAIL_TEMPLATE_ID="valid email_template_id"
-export SMS_TEMPLATE_ID="valid sms_template_id"
-export LETTER_TEMPLATE_ID="valid letter_template_id"
-export EMAIL_REPLY_TO_ID="valid email reply to id"
-export SMS_SENDER_ID="valid sms_sender_id - to test sending to a receiving number, so needs to be a valid number"
-export API_SENDING_KEY="API_team_key for sending a SMS to a receiving number"
-export INBOUND_SMS_QUERY_KEY="API_test_key to get received text messages"
-```
-
 To run the integration tests:
```

Comment:
To run the integration tests we need to get the environment variables from notify-pass, then copy them to `docker.env` or perhaps we use a different env file in `/scripts/run_with_docker.sh` that we add to .gitignore. 
Do we want to include instructions for this step?

Reply:
Yeah we do. I've moved them [a bit earlier](https://github.com/alphagov/notifications-ruby-client/pull/122/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R17), which is similar to where they are in the READMEs for our other repos. The instructions now tell you to make a `docker.env` file, instead of exporting a load of stuff.

Does that help?

Reply:
In order for the integration tests to run we need to set the properties which are in notify-pass credentials/client-integration-tests. So if I want to test a change in the client against I'd need to have set them to actual values that work. Happy to chat if I'm not making sense here.

Reply:
Ahh, I get what you mean 👍. Yes, that's definitely worth adding. In doing so, I realised the `docker.env` approach makes the setup harder, so I've pivoted to the `environment.sh` approach instead.

I've made those changes, together with a bit more justification, in a new commit. One disadvantage is that people will have to change their local setup to create the `environment.sh` file (it's easy, though).

Do you want to take another look?